corehq/apps/app_manager/models.py

```diff
@@ -5528,7 +5528,7 @@ class SavedAppBuild(ApplicationBase):
     def to_saved_build_json(self, timezone):
         data = super(SavedAppBuild, self).to_json().copy()
         for key in ('modules', 'user_registration', 'external_blobs',
-                    '_attachments', 'profile', 'translations'
+                    '_attachments', 'profile', 'translations',
```

Comment:
@dannyroberts you might be interested in this :)

Reply:
:scream:

Reply:
Wow, this error managed to stay in here for a full 5 years! 🎂https://github.com/dimagi/commcare-hq/commit/10876798178065eb483230a7502b9f1d5e31d0f9. Good catch @mkangia!

Reply:
quite a survivor!